librz/core/canalysis.c

```diff
@@ -4771,6 +4771,10 @@ static bool isSkippable(RzBinSymbol *s) {
 	return false;
 }
 
+static bool has_old_analysis_value(RZ_BORROW RZ_NONNULL RzCore *core) {
```

Comment:
I am not sure `RZ_BORROW/RZ_NONNULL` here are necessary, it's very short and static function.